rsqueakvm/main.py

```diff
@@ -66,6 +66,8 @@ def _usage(argv):
           Execution:
             -r|--run <code>  - Code will be compiled and executed in
                                headless mode, result printed.
+            -rr <code>       - Code will be compiled and executed in
```

Comment:
This is a bit of a hack: with the dynamic frame size calculation, we must not JIT a method that we don't know the frame size beforehand, so doits are never jitted. -rr just calls the passed code twice, so the second time we can jit.